test/test_nn.py

```diff
@@ -9340,14 +9341,32 @@ def test_conv_transposed_large(self, device):
         self.assertEqual(maxdiff2, 0)
         self.assertEqual(maxdiff3, 0)
 
+    @skipIfRocm
     @unittest.skipIf(not TEST_LARGE_TENSOR, "not enough memory to run test")
     def test_conv_large(self, device):
         dtype = torch.half if self.device_type == 'cuda' else torch.float
-        conv1 = nn.Conv2d(2, 2, 8, 8).to(device).to(dtype)
+        conv = nn.Conv2d(2, 2, 8, 8, bias=False).to(device).to(dtype)
         input_large = torch.randn(4096, 2, 512, 512, dtype=dtype, device=device)
-        ret = conv1(input_large)
-        self.assertEqual(ret[:2048], conv1(input_large[:2048]))
-        self.assertEqual(ret[2048:], conv1(input_large[2048:]))
+        # forward
+        ret = conv(input_large)
```

Comment:
I suspect this is splitting the input into equal chunks so there's no tail - can you make it a test that has unequal split?

Reply:
Sure I will

Reply:
I changed 4096 to 4097. Waiting for the CI status.